webpack/webpack.config.js

```diff
@@ -117,6 +130,38 @@ function createBundleAnalyzer() {
 	} );
 }
 
+function addYoastExternals( config, base, plugins, pluginVersionSlug ) {
```

Comment:
Looking a bit further at how Gutenberg handles this I think we can do this much more elegantly.

If we use a webpack config with the library option as WP uses: https://github.com/WordPress/gutenberg/blob/master/webpack.config.js#L73

We can point the entries directly at the node_modules entry files and don't have to include any local scripts nor handle assigning them to the window ourselves. That'd remove this complexity of generating webpack configs and remove all those files from the externals directory.

Reply:
Ah yes, this would be even more preferable.